
bool is_map_building_enabled
bool is_localization_enabled

int32 localization_quality

int32 board_temperature

int32 battery_percentage
bool is_dc_in
bool is_charging
